3.0.0.md

```diff
@@ -0,0 +1,209 @@
+# Upgrading Parse Server to version 3.0.0
+
+With the 3.0.0, parse-server has completely revamped it's cloud code interface. Gone are the backbone style calls, welcome promises and async await.
+
+In order to leverage those new nodejs features, you'll need to run at least node 8.10. We've dropped the support of node 6 a few months ago, so you should feed really comfortable at home.
```

Comment:
"so you should feed really comfortable at home." is a typo.  maybe:  "so if you haven't made the change yet, now would be a really good time to take the plunge."